MeetupPlatform.Api/Features/Meetups/GetMeetups/Filter/ConfigurationQuery/MeetupsFilterHelper.cs

```diff
@@ -0,0 +1,94 @@
+﻿namespace MeetupPlatform.Api.Features.Meetups.GetMeetups.Filter.ConfigurationQuery;
+
+using MeetupPlatform.Api.Domain;
+using MeetupPlatform.Api.Features.Meetups.GetMeetups.Filter.FilterSettings;
+using System.Linq;
+
+public static class MeetupsFilterHelper
+{
+    public static IQueryable<Meetup> GetMeetupsFilteredByFilterSettings(
+        this IQueryable<Meetup> meetupsQuery,
+        MeetupsFilterSettings meetupFilterSettings)
+    {
+        meetupsQuery = meetupsQuery.Filter(meetupFilterSettings);
+
+        meetupsQuery = meetupsQuery.SortSelection(meetupFilterSettings.SortOption);
+
+        return meetupsQuery.GetDataByPaginationSettings(meetupFilterSettings);
+    }
+
+    private static IQueryable<Meetup> GetDataByPaginationSettings(
+        this IQueryable<Meetup> meetupsQuery,
+        MeetupsFilterSettings meetupFilterSettings)
+    {
+        var skippedMeetups = (meetupFilterSettings.PageNumber - 1) * meetupFilterSettings.PageSize;
+
+        return meetupsQuery
+                .Skip(skippedMeetups)
+                .Take(meetupFilterSettings.PageSize);
+    }
+
+    private static IQueryable<Meetup> SortSelection(this IQueryable<Meetup> meetupsQuery, SortOptions sortOptions) => sortOptions switch
```

Comment:
Incorrect parameter name. In the `sortOptions` variable You only have 1 option, but variable name is plural.